src/models/applicationState.ts

```diff
@@ -229,6 +229,7 @@ export interface IAsset {
     format?: string;
     timestamp?: number;
     parent?: IAsset;
+    hasError?: boolean;
```

Comment:
I don't think we need to introduce a new modal property here that is persisted since the error may be transient.  If we REALLY wanted to store something then I would recommend we extend the `AssetState` to include a new enum for `Error`.